src/leappto/actor_cli.py

```diff
@@ -0,0 +1,154 @@
+    migrate_cmd.add_argument(
+        '--tcp-port',
+        default=None,
+        dest="forwarded_tcp_ports",
+        nargs='*',
+        type=_port_spec,
+        help='(Re)define target tcp ports to forward to macrocontainer - [target_port:source_port]'
+    )
+    migrate_cmd.add_argument(
+        '--no-tcp-port',
+        default=None,
+        dest="excluded_tcp_ports",
+        nargs='*',
+        type=_port_spec,
+        help='define tcp ports which will be excluded from the mapped ports [[target_port]:source_port>]'
+    )
+
+    migrate_cmd.add_argument(
+        '--exclude-path',
+        default=None,
+        dest="excluded_paths",
+        nargs='*',
+        type=_path_spec,
+        help='define paths which will be excluded from the source'
+    )
+    migrate_cmd.add_argument("--ignore-default-port-map", default=False,
+                             help='Default port mapping detected by leapp toll will be ignored', action="store_true")
+    migrate_cmd.add_argument('--container-name', '-n', default=None,
+                             help='Name of new container created on target host')
+    migrate_cmd.add_argument(
+        '--force-create',
+        action='store_true',
+        help='force creation of new target container, even if one already exists'
+    )
+    migrate_cmd.add_argument('--disable-start', dest='disable_start', default=False,
+                             help='Migrated container will not be started immediately', action="store_true")
+    _add_identity_options(migrate_cmd, context='source')
+    _add_identity_options(migrate_cmd, context='target')
+
+    return ap
+
+
+def make_base_string(s):
+    return {"value": s}
+
+
+def make_base_bool(s):
+    return {"value": s}
+
+
+def start_agent_if_not_available():
+    if 'SSH_AUTH_SOCK' in os.environ:
+        return
+    agent_details = subprocess.check_output(["ssh-agent", "-c"], stderr=PIPE).splitlines()
+    agent_socket = agent_details[0].split()[2].rstrip(";")
+    agent_pid = agent_details[1].split()[2].rstrip(";")
+    os.environ["SSH_AUTH_SOCK"] = agent_socket
+    os.environ["SSH_AGENT_PID"] = agent_pid
+
+
+def migrate_machine(arguments):
+    loader.load(ACTOR_DIRECTORY)
+    data = {
+        "target_host": make_base_string(arguments.target),
+        "source_host": make_base_string(arguments.machine),
+        "tcp_port_list": {"tcp": to_port_map(arguments.forwarded_tcp_ports)},
+        "excluded_tcp_port_list": {"tcp": map(lambda x: int(x[0]), arguments.excluded_tcp_ports)},
+        "excluded_paths": {"value": arguments.excluded_paths},
+        "start_container": make_base_bool(not arguments.disable_start),
+        "target_user_name": make_base_string(arguments.target_user),
+        "source_user_name": make_base_string(arguments.source_user),
+    }
+
+    if not registry.get_actor('migrate-machine').execute(data):
+        pprint(data)
+        exit(-1)
```

Comment:
don't we want some message on this exit?

Reply:
i mean more informative that just that preety print

Reply:
Since this is just WIP this has not been finished yet - That's what needs to be properly done